lib/widgets/compose_box.dart

```diff
@@ -735,7 +736,23 @@ class _SendButtonState extends State<_SendButton> {
 
     final store = PerAccountStoreWidget.of(context);
     final content = widget.contentController.textNormalized;
-    store.sendMessage(destination: widget.getDestination(), content: content);
+
+    try {
+      // TODO(#720) put input(s) and send button into a disabled "working on it"
+      //   state (allowing input text to be selected for copying).
+      await store.sendMessage(destination: widget.getDestination(), content: content);
```

Comment:
This changes more than the error behavior — it also means we wait to clear the content input until the request completes.

Our promptly clearing the input isn't entirely a good thing; that's what #720 is about. But if we take that out without doing the other work called for in #720, I think the effect is that there's a lack of feedback so that it feels like the send button didn't do anything, or is sluggish.

Trying this, I notice that even on my fast home network connection. Then to demonstrate how it behaves when the request is especially slow, try this patch:
```diff
--- lib/api/core.dart
+++ lib/api/core.dart
@@ -188,6 +188,7 @@ class ApiConnection {
     if (params != null) {
       request.bodyFields = encodeParameters(params)!;
     }
+    await Future<void>.delayed(const Duration(seconds: 2));
     return send(routeName, fromJson, request, overrideUserAgent: overrideUserAgent);
   }
 
```

The way to fix that looks something like:
```dart
final sendFuture = store.sendMessage(…);
widget.contentController.clear();

try {
  await sendFuture;
} on …
```

Or I guess more straightforwardly, we can just call `clear` before `sendMessage`.

Reply:
Makes sense.